content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+## Bioactivity data profile
+
+[TOC]
+
+
+
+## Main Objective
+
+To provide guidance on the minimum set of metadata and semantics required to describe any chemical biology experiments, and how to translate bioactivity data to CHEMBL standards. This recipe also applies to binding, functional and ADME or toxicological assay types.
+
+
+## Graphical overview of the Recipe FAIRification Objectives
+
+```mermaid
+graph TD
+    A[Primary data at one concentration]-->B{Number of readouts per molecules?}
+    B-->|one|V{Which unit?}
+    V-->|percentage|D
+    V-->|value|F
+    B-->|multi|C[valid ones]
+    C-->|experiments|D{if triplicates}
+    D-->|aggregating or filtering|E[validated ones]
+    E--> |calculating|G[One average readout value <br> per concentration]
+    G-->F[CC/AC/IC50/IC90 curves]
+```
```

Comment:
```suggestion
the diagram should be explained in more details:

-  provide a definition of 'readout',
- clarify decision box (for instance what is the difference between `value` and `percentage` in terms of meaning? when `readout` is of type `multi`, does it mean there is no check on value or unit?
- what is the meaning of the `experiment` edge in this context?
- `if triplicate` box : what happens if the condition is not met?

Reply:
@agiani99  do you mind taking this one? Thanks!